PhotoKeeper/PhotoPreviewViewController.swift

```diff
@@ -8,71 +8,71 @@
 
 import UIKit
 
-class PhotoPreviewViewController: UIViewController, UITextFieldDelegate, UITextViewDelegate{
+class PhotoPreviewViewController: UIViewController, UITextFieldDelegate, UITextViewDelegate {
     // MARK: UI Stuff
     let store = PhotoStore.shared
     var image = UIImage()
     let metaForm = UIView()
     let titleField = UITextField()
-    let descriptionField = UITextView()
+    let descField = UITextView()
     let submitButton = UIButton()
     let iView = UIImageView()
-    
+
     let blueblack = UIColor(hue: 0.6333, saturation: 0.15, brightness: 0.64, alpha: 1.0)
     lazy var blueblackCG = blueblack.cgColor
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        
         setupView()
         handleKeyboardEvents()
-
     }
-    
+
     func setupView() {
         view.backgroundColor = .black
-        let bg = UIImageView()
-        bg.contentMode = .scaleAspectFill
-        bg.image = image
-        bg.frame = UIScreen.main.bounds
-        bg.frame = CGRect(x: 0, y: 0, width: view.frame.width, height: view.frame.height)
-        bg.clipsToBounds = true
-    
-        view.addSubview(bg)
+        let bgView = UIImageView()
+        bgView.contentMode = .scaleAspectFill
+        bgView.image = image
+        bgView.frame = UIScreen.main.bounds
+        bgView.frame = CGRect(x: 0, y: 0, width: view.frame.width, height: view.frame.height)
```

Comment:
you're setting frame twice here. Both appear to be identical but setting to `bounds` is the simplest